BUILDSCRIPT_MULTIPROC.bash44

```diff
@@ -1,20 +1,13 @@
 #!/bin/bash
 
-CONFIG="default"
-while [ $# -gt 0 ] ; do
-    if [ "$1" = "--config" ] && [ $# -gt 1 ] ; then
-        CONFIG="$2"
-        shift
-    fi
-    shift
-done
-
 CXX="${CXX:-g++}"
-CXXFLAGS="${CXXFLAGS:--Wall -Wextra -Wshadow -Wold-style-cast -Woverloaded-virtual -pedantic -O3 -flto}"
+CXXFLAGS="${CXXFLAGS:--Wall -Wextra -Wshadow -Wold-style-cast -Woverloaded-virtual -pedantic -O3}"
+CXXLTOFLAGS="${CXXLTOFLAGS:--flto}"
```

Comment:
Not all platforms/environments support link time optimization. Particularly relevant to me is that it isn't supported for Darwin in `nixpkgs`. As such, I think it would be nice to have a separate environment variable to override, so that folks that want to disable link time optimization don't have to override `$CXXFLAGS` just to remove that one option.